editor/editor_inspector.cpp

```diff
@@ -1709,6 +1709,8 @@ void EditorInspector::_edit_set(const String &p_name, const Variant &p_value, bo
 		}
 	}
 
+	changing++;
```

Comment:
I moved this variable here to fix multiedit node and all other flows we can have inside this logic too.